protocol/test/forge/v2/safeGuards/ThresholdSafeGuard.t.sol

```diff
@@ -404,6 +401,35 @@ contract ThresholdSafeGuardTest is ThresholdSafeGuardTestBase {
         doCheck("", safeOwners[1], signature);
     }
 
+    function test_checkTransaction_withOverrideTresholdSet_emptyData() public {
+        vm.startPrank(executor);
+        bytes4 fnSelector = bytes4(0);
+
+        guard.setFunctionThreshold(address(mock), fnSelector, 3);
+
+        // Add a couple more executors to the safe which we can sign on behalf of.
+        (uint256 pk2, uint256 pk3) = addSafeOwners();
+
+        // An empty function call (eg ETH transfer)
+        bytes memory fnCall = bytes("");
```

Comment:
can you do an explicit ETH transfer and test amounts? receiver should have `fallback` or `receive`

Reply:
great idea - i've done a full safe execute for the full e2e flow

Reply:
Fixed in https://github.com/TempleDAO/temple/pull/814/commits/3933dcacf45c04e2bb680b56d4921a0d6b6ba65a